src/homework07/Transport/CivilTransport.java

```diff
@@ -0,0 +1,36 @@
+package homework07.Transport;
+
+public class CivilTransport extends AirTransport {
+    protected int countPassengers;
+    protected boolean businessClass;
+
+    public CivilTransport(int countPassengers, boolean businessClass, int wingspan, int minDistanceForTakeoff, int horsePower, int maxSpeed, int weightKG, String brand) {
+        super(wingspan, minDistanceForTakeoff, horsePower, maxSpeed, weightKG, brand);
+        this.countPassengers = countPassengers;
+        this.businessClass = businessClass;
+    }
+
+    @Override
+    public String toString() {
+        return "СivilTransport{" +
+                "businessClass=" + businessClass +
+                ", wingspan=" + wingspan +
+                ", minDistanceForTakeoff=" + minDistanceForTakeoff +
+                ", horsePower=" + horsePower +
+                ", maxSpeed=" + maxSpeed +
+                ", weightKG=" + weightKG +
+                ", brand='" + brand + '\'' +
+                ", kWPower=" + getkWPower() + '}';
+    }
+
+    public double getkWPower() {
```

Comment:
getKWPower will be correct naming due to camelcase

Reply:
thanks